### PR TITLE
Change data classes' base from TNamed to TObject

### PR DESCRIPTION
**Pull request by alekgajos**
additionally, removed obsolete "charge" option from JPetSigCh.

Note: unit test of JPetReader requires update of a data file "timewindows.root" at Sphinx. I have temporarily substituted this file and UTs at Travis pass correctly (UTs also pass locally on my machine).
Now the file is back to the old version at Sphinx. I will update it permanently if we decide to merge this PR.

### CODE REVIEW

File: JPetPhysSignal/JPetPhysSignal.cpp [hunk 1]
```diff
@@ -23,7 +23,7 @@ JPetPhysSignal::JPetPhysSignal() :
   fPhe(0),
   fQualityOfPhe(0)
 {
-  SetNameTitle("JPetPhysSignal", "Physical signal structure");
+  //  SetNameTitle("JPetPhysSignal", "Physical signal structure");
```
Comment (wkrzemien) [inline JPetPhysSignal/JPetPhysSignal.cpp]: Remove it, instead of comment out

File: JPetRawSignal/JPetRawSignal.h [hunk 2]
```diff
@@ -103,9 +102,8 @@ class JPetRawSignal: public JPetBaseSignal
 private:
   std::vector<JPetSigCh> fLeadingPoints; ///< vector of JPetSigCh objects from leading edge of the signal
   std::vector<JPetSigCh> fTrailingPoints; ///< vector of JPetSigCh objects from trailing edge of the signal
-  JPetSigCh fTOTPoint;
 
-ClassDef(JPetRawSignal, 3)
+ClassDef(JPetRawSignal, 5)
```
Comment (wkrzemien) [inline JPetRawSignal/JPetRawSignal.h]: JPetRawSignal,4 ?

File: JPetSigCh/JPetSigCh.cpp [hunk 3]
```diff
@@ -23,7 +23,7 @@ ClassImp(JPetSigCh);
 const float JPetSigCh::kUnset = std::numeric_limits<float>::infinity();
 
 void JPetSigCh::Init() {
-  SetNameTitle("JPetSigCh", "Signal Channel Structure");
+  //  SetNameTitle("JPetSigCh", "Signal Channel Structure");
```
Comment (wkrzemien) [inline JPetSigCh/JPetSigCh.cpp]: remove it

File: JPetTimeWindow/JPetTimeWindow.h [hunk 4]
```diff
@@ -18,21 +18,19 @@
...
  * A single TSlot contains many SigCh objects representing TDC hits recorded during a time window of the TRB board.
  */
-class JPetTimeWindow: public TNamed
+class JPetTimeWindow: public TObject
 {
 public:
-/// @todo think about changing TClonesArray to something else ? what about cleaning
   JPetTimeWindow() {
-    SetName("JPetTimeWindow");
+    //    SetName("JPetTimeWindow");
```
Comment (wkrzemien) [inline JPetTimeWindow/JPetTimeWindow.h]: remove it

**Comment by alekgajos** [comment]
Yes, the test file timewindows.root is used by both master and develop versions. I have reverted this file version at Sphinx to the old version (correct for the master branch) soon after writing that previous comment so UT-s should not be broken. 

The new file (relevant for thie PR) is available at Sphinx as "timewindows.root.new" so that it can be quickly substituted.

**Comment by wkrzemien** [comment]
Ok. Please check the Unpacker changes which can be seen in Files changes and remove this commented code and I will accept it

**Pull request merged**